### PR TITLE
Small text improvements

### PR DESCRIPTION
**Pull request by jgardiner68**
* small textual improvements to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,12 +290,12 @@ These configuration elements are enough for most connection scenarios. Now we wi
...
 ### 1.4.2. Configuring Hazelcast Python Client
 
-Hazelcast Python client can be configured programmatically.
+The Hazelcast Python client can be configured programmatically.
 
 This section describes some network configuration settings to cover common use cases in connecting the client to a cluster. See the [Configuration Overview section](#3-configuration-overview)
 and the following sections for information about detailed network configurations and/or additional features of Hazelcast Python client configuration.
 
-An easy way to configure your Hazelcast Python client is to create a `ClientConfig` object and set the appropriate options. Then you can
+An easy way to configure your Hazelcast Python client is to create a `ClientConfig` object and set the appropriate config options. Then you can
```
Comment (nfrankel) [inline README.md]: An easy way to configure your Hazelcast Python client is to create a `ClientConfig` object and set the appropriate configuration options. Then you can
supply this object to your client at the startup. This is the programmatic configuration approach.

File: README.md [hunk 2]
```diff
@@ -2215,7 +2215,7 @@ Map entries in Hazelcast are partitioned across the cluster members. Hazelcast c
 
 These benefits do not come for free, please consider the following trade-offs:
 
-- Clients with a Near Cache will have to hold the extra cached data, which increases memory consumption.
+- Clients with a Near Cache will have to hold the extra cached data, which increases their memory consumption.
```
Comment (nfrankel) [inline README.md]: Clients with a Near Cache will have to hold the extra cached data, which increases their respective memory consumption.

File: README.md [hunk 3]
```diff
@@ -311,8 +311,8 @@ client = hazelcast.HazelcastClient(config)
 
 ---
 
-If you run the Hazelcast IMDG members in a different server than the client, you most probably have configured the members' ports and cluster
-names as explained in the previous section. If you did, then you need to make certain changes to the network settings of your client.
+If you run the Hazelcast IMDG members on a different server than the client, you most probably have configured the members' ports and cluster
+names as explained in the previous section. If you did, then you need to make match those changes to the network settings of your client.
```
Comment (nfrankel) [inline README.md]: If you run the Hazelcast IMDG members on a different server than the client, you most probably have configured the members' ports and cluster
names as explained in the previous section. If you did, then you need to make certain changes to the network settings of your client.
names as explained in the previous section. If you did, then you need to match those changes to the network settings of your client.

**Comment by mdumandag** [comment]
Closing in favor of #466

**Pull request closed without merging**